OpenRA.Mods.Common/Traits/Buildings/Bridge.cs

```diff
@@ -47,7 +47,18 @@ class BridgeInfo : ITraitInfo, IRulesetLoaded, Requires<HealthInfo>, Requires<Bu
 
 		public object Create(ActorInitializer init) { return new Bridge(init.Self, this); }
 
-		public void RulesetLoaded(Ruleset rules, ActorInfo ai) { DemolishWeaponInfo = rules.Weapons[DemolishWeapon.ToLowerInvariant()]; }
+		public void RulesetLoaded(Ruleset rules, ActorInfo ai)
+		{
+			if (string.IsNullOrWhiteSpace(DemolishWeapon))
+				throw new YamlException("Invalid value found for DemolishWeapon of {0}'s Bridge trait.".F(ai.Name));
+
+			WeaponInfo weapon;
+			var weaponToLower = DemolishWeapon.ToLowerInvariant();
```

Comment:
I am using `weaponToLower` for consistency with other areas of code doing the same work.